sentry_dart_plugin/bin/sentry_dart_plugin.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:sentry_dart_plugin/sentry_dart_plugin.dart';
+
+void main(List<String> arguments) {
+  SentryDartPlugin().run(arguments);
```

Comment:
Can we use the dart client SDK to capture exceptions in this CLI? 😁

Reply:
good catch

Reply:
yep, we can do that too, will figure out a DSN for that, we do in another PR though.

I wanted to do the same on the gradle plugin, https://github.com/getsentry/sentry-android-gradle-plugin/pull/126
but we need to have an opt-out option.